src/client/datascience/kernel-launcher/kernelProcess.ts

```diff
@@ -191,29 +190,7 @@ export class KernelProcess implements IKernelProcess {
                     return;
                 }
                 traceError('Kernel died', error, stderr);
-                if (error instanceof PythonKernelDiedError) {
```

Comment:
This error is never thrown anywhere, it was only thrown in the case of kernels being launched using daemon (which isn't used anymore)